src/components/AnnouncementCard/AnnouncementCard.jsx

```diff
@@ -24,17 +25,35 @@ const styleDate = (date) => {
    return `${month} ${day}, ${year}`;
 };
 
-export default (Announcement) => (
-   <Card style={{ width: '18rem ' }}>
-      <Card.Body>
-         <div>
-            <Card.Title>{Announcement.Announcement.title}</Card.Title>
-            <Card.Subtitle className="mb-2 text-muted">
-               {styleDate(Announcement.Announcement.date)}
-            </Card.Subtitle>
-            <Card.Text>{Announcement.Announcement.text}</Card.Text>
-         </div>
-         <Card.Link href={Announcement.Announcement.link}>Read More</Card.Link>
-      </Card.Body>
-   </Card>
-);
+export default (Announcement) => {
+   const [showModal, setShowModal] = useState('false');
+   const handleCloseModal = () => {
+      console.log('handle close');
+      setShowModal('false');
+   };
+   const handleShowModal = () => {
+      console.log('handle show');
+      setShowModal('true');
+   };
```

Comment:
The reason why the Modal isn't showing/hiding is because you set state to the string 'false' not the boolean value false. Also you can probably consolidate the handleCloseModal and handleShowModal into a single function.